README.md

```diff
@@ -1,11 +1,52 @@
-# solution-pack-soc-simulator
+# SOC Simulator Solution Pack
+
+## Overview
+
+### Introduction
 
 The FortiSOAR™ SOC Simulator Solution Pack (solution-pack-soc-simulator). This solution pack enables users to experience the power and capability of FortiSOAR™ incident response for SOC Simulator scenarios. This pack contains simulation data and utilities for demonstrating FortiSOAR capabilities around a variety of important SOC use-cases without having to integrate with actual device endpoints.
 
-## Deploying SOC Simulator Solution Pack
+### Usage
+
+More information about the usage of the SOC Simulator Solution Pack can be found [here](https://github.com/fortinet-fortisoar/solution-pack-soc-simulator/blob/develop/docs/solution-pack-guide.md).
+
+## Version Information
+
+- Solution Pack Version: 1.0.
+- FortiSOAR™ Version Tested on: 7.2.0
+- Authored By: Fortinet
+- Certified: Yes
+
+## Prerequisite
+
+- Before you deploy the SOC Simulator Solution Pack, ensure that you have deployed the SOAR Framework Solution Pack([solution-pack-soar-framework](https://github.com/fortinet-fortisoar/solution-pack-soar-framework)). The steps for deploying a solution pack are mentioned in the [Deploying a Solution Pack](https://github.com/fortinet-fortisoar/how-tos/blob/main/DeployingASolutionPack.md) article. 
+- Ensure that the following URLs are unblocked in your environment: 
```

Comment:
Why following URLs needs to be unblocked and what is a use of it?

Reply:
We need to mention this in doc for  customers.

Reply:
> Why following URLs needs to be unblocked and what is a use of it?

I looked into the connector code and updated the URL accordingly. These URLs are used to gather threat intel data. Updated the same in documentation as well